GeosCore/drydep_mod.F90

```diff
@@ -479,7 +479,7 @@ SUBROUTINE UPDATE_DRYDEPFREQ( Input_Opt, State_Chm, State_Diag, State_Grid, &
           !-----------------------------------------------------------
           ! Special treatment for snow vs. ice
```

Comment:
Given this section treats snow and ice the same I think we should update this comment to be "Special treatment for snow and ice". Having "vs" makes it look like they are treated differently.

Reply:
Agree. The comment felt a bit ambiguous to me so I left it as is in case anyone more knowledgeable about this specific case wanted to suggest a different approach.  Based on how `IsSnow` used to be defined it will have always treated snow and ice the same until 13.4 so my gut feeling is that it should read 

> Special treatment for snow and ice 

as you suggested

Reply:
This comment is now fixed in c23b1efbe.